pkg/backend-storage/backend-storage.go

```diff
@@ -65,11 +77,63 @@ func CreateIfNeeded(vmi *corev1.VirtualMachineInstance, clusterConfig *virtconfi
 			VolumeMode:       &modeFile,
 		},
 	}
+	// Whatever owns the VMI (i.e. a VM) now owns the PVC
+	for _, ownerRef := range vmi.ObjectMeta.OwnerReferences {
+		newRef := ownerRef.DeepCopy()
+		pvc.ObjectMeta.OwnerReferences = append(pvc.ObjectMeta.OwnerReferences, *newRef)
+	}
+
+	pvc, err = client.CoreV1().PersistentVolumeClaims(vmi.Namespace).Create(context.Background(), pvc, metav1.CreateOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	return pvc.ObjectMeta.Name, nil
+}
 
-	_, err = client.CoreV1().PersistentVolumeClaims(vmi.Namespace).Create(context.Background(), pvc, metav1.CreateOptions{})
+func RemoveOldPVCFor(vmi string, pod *v1.Pod, client kubecli.KubevirtClient) {
```

Comment:
Which PVCs is this function supposed to remove? The backed PVC for the TPM?

Reply:
The backend PVC used by the VMI pre-migration.

Reply:
The maybe we should rename the function to something like `RemoveOldBackendPVCFor`